src/prototypo-canvas.js

```diff
@@ -202,6 +210,7 @@ function drawTypographicFrame() {
 	this.typographicFrame.xHeight.size.height = 1 / this.zoom;
 	this.typographicFrame.capHeight.size.height = 1 / this.zoom;
 	this.typographicFrame.linearDraggingHelper ? this.typographicFrame.linearDraggingHelper.strokeWidth = 2 / this.zoom : null;
+	this.typographicFrame.snappingHelper ? this.typographicFrame.snappingHelper.strokeWidth = 2 / this.zoom : null;
 	this.typographicFrame.linearDraggingHelper ? this.typographicFrame.linearDraggingHelper.dashArray = [ 8 / this.zoom, 8 / this.zoom ] : null;
```

Comment:
😅 At least in my mind I was always right, that may have been a typo I copied several times thinking it was the right syntax. I'll fix it!